charts/kong/ci/admin-api-service-clusterip-values.yaml

```diff
@@ -16,3 +16,17 @@ dblessConfig:
 
 ingressController:
   enabled: false
+
+# NOTE: The purpose of those values below is to:
+# - to speed up, the test
+# - to prevent anonymous reports from being sent
+readinessProbe:
+  httpGet:
+    path: "/status"
+    port: status
```

Comment:
@tao12345666333 The reason for this is that we don't want to use the status config readiness endpoint here because the ingress controller is not running so it won't send the config. 

OTOH though it was like this before so I'm not sure ATM why that worked 🤔 . Let's revert this

Reply:
https://github.com/Kong/charts/actions/runs/9547976613/job/26315250672

```
Notice: 2024/06/17 13:25:50 [notice] 2405#0: *553 [lua] ready.lua:118: fn(): not ready for proxying: no configuration available (empty configuration present), client: 10.244.0.1, server: kong_status, request: "GET /status/ready HTTP/1.1", host: "10.244.0.20:8100"
```